.github/workflows/ci.yml

```diff
@@ -25,7 +26,7 @@ jobs:
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
       mysql:
-        image: mysql:8.0
+        image: mariadb
```

Comment:
Is this mysql:8.0 issue still existing?

Reply:
I tried locally yesterday night, sadly yes, the linked issue it this one:

https://github.com/launchbadge/sqlx/issues/846

Reply:
Also, if @Notmeor is right the linked issue in webpki (https://github.com/briansmith/webpki/issues/54) is open since 2017 and is currently affecting a lot of the rust ecosystem.

Reply:
Get it.